### DISCUSSION TITLE
Next botorch release

### DISCUSSION THREAD

**Discussion by jduerholt** [meta-pytorch/botorch#1683] [General]
Hi botorch developers,

I asked myself if you already know when the next release will happen, as there are now features in the most recent main which are of interest for our code based on botorch. If the next release will be there soon, it would not make sense to change our pipelines to the recent main, but if it will take some more time (which is totally fine), I would update our internal pipelines ;)

Best,

Johannes

**Comment by saitcakmak** [comment]
I am checking with internal folks to see if there's anything blocking us on a release. My guess is we can put out a release by the end of next week.

**Reply by jduerholt** [reply]
Thanks for the release!
